cwms-data-api/src/test/java/cwms/cda/api/ForecastInstanceControllerTestIT.java

```diff
@@ -150,55 +300,69 @@ void test_create_get_delete_get() throws IOException {
+            .redirects().max(3)
+            .post(PATH)
+        .then()
+            .log().ifValidationFails(LogDetail.ALL,true)
+        .assertThat()
+            .statusCode(is(HttpServletResponse.SC_CREATED));
 
         // Step 2)
         // Retrieve the inst and assert that it exists
         given()
-                .log().ifValidationFails(LogDetail.ALL,true)
-                .accept(Formats.JSONV2)
-                .queryParam(Controllers.OFFICE, OFFICE)
-                .queryParam(Controllers.NAME, SPEC_ID)
-                .queryParam(Controllers.LOCATION_ID, locationId)
-                .queryParam(Controllers.FORECAST_DATE, forecastDate)
-                .queryParam(Controllers.ISSUE_DATE, issueDate)
-                .when()
-                .redirects().follow(true)
-                .redirects().max(3)
-                .get(PATH)
-                .then()
-                .log().ifValidationFails(LogDetail.ALL,true)
-                .assertThat()
-                .statusCode(is(HttpServletResponse.SC_OK))
-                .body("spec-id", equalTo("test-spec"))
-                .body("date-time", equalTo(1624284010000L))
-                .body("issue-date-time", equalTo(1653221020000L))
-                .body("first-date-time", equalTo(1692702150000L))
-                .body("last-date-time", equalTo(1727017260000L))
-                .body("max-age", equalTo(5))
-                .body("time-series-count", equalTo(3))
-                .body("notes", equalTo("test notes"))
-                .body("metadata.key1", equalTo("value1"))
-                .body("metadata.key2", equalTo("value2"))
-                .body("metadata.key3", equalTo("value3"))
-                .body("filename", equalTo("testFilename.txt"))
-                .body("file-description", equalTo( "test file description"))
-                .body("file-data", equalTo("dGVzdCBmaWxlIGNvbnRlbnQ="))
+            .log().ifValidationFails(LogDetail.ALL,true)
+            .accept(Formats.JSONV2)
+            .queryParam(Controllers.OFFICE, OFFICE)
+            .queryParam(Controllers.DESIGNATOR, designator)
+            .queryParam(Controllers.FORECAST_DATE, forecastDate)
+            .queryParam(Controllers.ISSUE_DATE, issueDate)
+        .when()
+            .redirects().follow(true)
+            .redirects().max(3)
+            .get(PATH + SPEC_ID)
+        .then()
+            .log().ifValidationFails(LogDetail.ALL,true)
+        .assertThat()
+            .statusCode(is(HttpServletResponse.SC_OK))
+            .body("spec.spec-id", equalTo(SPEC_ID))
+            .body("date-time", equalTo(1624284000000L))
+            .body("issue-date-time", equalTo(1653220980000L))
+            .body("max-age", equalTo(5))
+            .body("notes", equalTo("test notes"))
+//                .body("metadata.key1", equalTo("value1"))
+//                .body("metadata.key2", equalTo("value2"))
+//                .body("metadata.key3", equalTo("value3"))
```

Comment:
is metadata going away or its tested somewhere else?

Reply:
It currently doesn't work in the schema: https://bitbucket.hecdev.net/projects/CWMS/repos/cwms_database/pull-requests/326/overview?commentId=17908